composer.json

```diff
@@ -16,10 +16,10 @@
         }
     ],
     "require": {
-        "php" : ">=5.3.0"
+        "php" : "^5.3|^7.0"
     },
     "require-dev": {
-        "phpunit/phpunit" : "4.*",
+        "phpunit/phpunit" : "^4.0|^5.0",
```

Comment:
I understand it does the same, yet it's [not standard](https://github.com/thephpleague/glide/blob/master/composer.json#L26) for league packages.

Also it's easier to bump version, e.g to 4.7 (due to prophecy integration) in the future.

Could you link me to some repos, where do you find these?